sdk/runanywhere-commons/scripts/build-ios.sh

```
@@ -622,6 +734,29 @@ create_backend_xcframework() {
             continue
         fi
 
+        # For MetalRT: strip sentencepiece's flag.cc.o to avoid duplicate _FLAGS_help with sherpa-onnx
+        if [[ "$BACKEND_NAME" == "metalrt" ]]; then
+            local TMPSTRIP
+            TMPSTRIP=$(mktemp -d) || { log_error "Failed to create temp directory"; exit 1; }
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)